go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/go-vela/server
 
+replace github.com/go-vela/types => github.com/JayCeeJr/types v0.9.1-beta
```

Comment:
Should this line be removed?

Looks like https://github.com/go-vela/types/pull/193 was merged 👍 

So I think you could vendor in the latest changes from that repo instead of this line.

Reply:
I'll pull that line out